routes/api/memes/index.js

```diff
@@ -0,0 +1,261 @@
+const express = require('express');
+const bcrypt = require('bcrypt');
+const models = require('../models');
+const auth = require('../auth');
+const router = express.Router();
+
+/**
+ * Creates meme
+ */
+router.post('/', auth.isAuthenticated, (req, res) => {
+   models.Meme.create({
+        title: req.body.title,
+        link: req.body.link,
+        creatorId: req.session.userId,
+        TemplateId: parseInt(req.body.templateId) || null,
+        CommunityId: parseInt(req.body.communityId) || null,
```

Comment:
Could you please add an explicit check to ensure that the passed in communityId exists and is an int? Sequelize apparently allows nulls here when we should force a meme to be part of a community.